demos/python_demos/face_recognition_demo/face_recognition_demo.py

```diff
@@ -70,7 +70,14 @@ def build_argparser():
                         help="Path to the Facial Landmarks Regression model XML file")
     models.add_argument('-m_reid', metavar="PATH", default="", required=True,
                         help="Path to the Face Reidentification model XML file")
-
+    models.add_argument('-fd_iw', '--fd_input_width', default=0, type=int,
+                         help="(optional) specified the input shape of detection model " \
+                         "(default: use default input shape of model). Both -fd_iw and -fd_ih parameters " \
+                         "should be specified for reshape.")
+    models.add_argument('-fd_ih', '--fd_input_height', default=0, type=int,
+                         help="(optional) specified the input shape of detection model " \
```

Comment:
* Following demo's style, it should be `help="(optional) Specify the input shape of detection model " \` or just `help="(optional) Height of detection model's input " \`. The same goes for `--fd_input_width`.

* Please, update `README.md` accordingly.

Reply:
ok. thanks